recipe/meta.yaml

```diff
@@ -25,35 +25,37 @@ build:
 requirements:
   host:
     - python
-    - debugpy >=1.0.0
-    - hatchling >=1.4
-    - ipython >=7.23.1
-    - jupyter_client >=6.1.12
-    - packaging
     - pip
-    - comm >=0.1
-    - psutil
+    - setuptools
     - wheel
+    - debugpy
+    - comm
+    - hatchling
+    - ipython
+    - jupyter_client
+    - packaging
+    - psutil
   run:
-    - appnope  # [osx]
-    - comm >=0.1.1
-    - debugpy >=1.0
+    - python
+    - debugpy >=1.6.5
     - ipython >=7.23.1
+    - comm >=0.1.1
+    - traitlets >=5.4.0
     - jupyter_client >=6.1.12
-    - matplotlib-inline >=0.1
     - nest-asyncio
-    - packaging
-    - psutil
-    - python
-    - pyzmq >=17
     - tornado >=6.1
-    - traitlets >=5.4.0
+    - matplotlib-inline >=0.1
+    - appnope  # [osx]
+    - pyzmq >=17
+    - psutil
+    - packaging
 
 test:
   requires:
+    - python
     - curio  # [not win]
     - flaky
-    - jupyter_client
+    - jupyter_client !=7.3.2=*_0
```

Comment:
Why is this version only excluded from tests?

Reply:
I think I got my wires crossed between `jupyter_client` and `jupyter_core`. I'll fix it.

Reply:
Also I was cruising conda-forge when I had some errors and came across this https://github.com/conda-forge/ipykernel-feedstock/blob/d5e8001e203cf2097e076debf20a67fb58ad2a49/recipe/meta.yaml#L75

Reply:
`jupyter_client` should still not be in the test requirements since it's also a run requirement

Reply:
It's been removed.